docs/next/guides/accessories-and-menus/keyboard-shortcuts.md

```diff
@@ -0,0 +1,282 @@
+- Switch to a different context: [`setActiveContextName(<name>)`](@/api/shortcutmanager.md#setactivecontextname)
+- Get an already-registered context: [`getContext(<name>)`](@/api/shortcutmanager.md#getcontext)
+- Create and register a new context: [`addContext(<name>)`](@/api/shortcutmanager.md#addcontext)
+
+For example, if you're using a complex [custom editor](@/guides/cell-functions/cell-editor.md##how-to-create-a-custom-editor), 
+creating a new shortcut context can let you navigate your editor's UI with arrow keys (normally used for navigating the grid).
+
+### Adding a custom keyboard shortcut
+
+To add a custom keyboard shortcut:
+1. Select a [context](#keyboard-shortcut-contexts) in which you want to add a shortcut, for example:
+    ```js
+    const gridContext = hot.getShortcutManager().getContext('grid');
+    ```
+2. Using the selected context's [`addShortcut()`](@/api/shortcutcontext.md#addshortcut) method, add your keyboard shortcut:
+    ```js
+    const gridContext = hot.getShortcutManager().getContext('grid');
+
+    gridContext.addShortcut({
+      group: 'group_ID',
+      keys: [['enter']],
+      callback: () => {},
+    });
+    ```
+    The [`keys`](@/api/shortcutcontext.md#addshortcut) parameter:
+    - Accepts all the [`KeyboardEvent.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) key names.
+    - Accepts key names in both lowercase and uppercase (e.g. both `Enter` and `enter` work)
+    - Handles key-name discrepancies between browsers (e.g. both `'Spacebar'` and `' '` work)
+    ::: tip
+    **Using the <kbd>Alt</kbd> (<kbd>⎇</kbd>) modifier key**
+    
+    <kbd>Alt</kbd> (<kbd>⎇</kbd>) is often used for typing special characters (e.g. letters wich diacritical marks),
+    and its behavior may vary depending on the user's language and keyboard settings.
+    
+    To properly use <kbd>Alt</kbd> (<kbd>⎇</kbd>) in your shortcut, you may need to pass language-specific signs (such as `à` or `ś`) to the [`keys`](@/api/shortcutcontext.md#addshortcut) parameter.
+    :::
+
+#### Setting the order of keyboard actions
+
+You can assign multiple actions to a single keyboard shortcut. By default, when you assign a new action, it runs after any already-assigned actions.
+
+To set your own order of actions, use the `position` and `relativeToGroup` parameters of the [`addShortcut()`](@/api/shortcutcontext.md#addshortcut) method:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({
+  group: 'customNumericEditor',
+  position: 'before',
+  relativeToGroup: 'editorManager.handlingEditor',
+  runOnlyIf: () => { hot.getSelected !== void 0 },
+  keys: [['F2']],
+  callback: () => {
+    if (hot.getActiveEditor().cellProperties.type === 'numeric') {
+      return false; // the `F2` shortcut won't work for `numeric` cells
+    }
+    
+    // another action
+  },
+});
+```
+
+#### Adding a conditional keyboard action
```

Comment:
Maybe it should be moved before above example? The `runOnlyIf` has been introduced before showing how it works.

Reply:
Sure, done: [c4a4d66](https://github.com/handsontable/handsontable/pull/9154/commits/c4a4d6628417f4fa33ef328c775f1775ccfa0dc1)